tests/test_locales.py

```diff
@@ -997,72 +997,77 @@ def test_format_relative_future(self):
         assert self.locale._format_timeframe("months", 2) == "2개월"
-
         assert self.locale._format_timeframe("year", 1) == "1년"
         assert self.locale._format_timeframe("years", 2) == "2년"
 
-
     def test_format_relative_now(self):
-
-        result = self.locale._format_relative("조금 전", "now", 0)
-
-        assert result == "조금 전"
+        assert self.locale._format_relative("지금", "now", 0) == "지금"
 
     def test_format_relative_past(self):
-
-        result = self.locale._format_relative("한시간", "hour", 1)
-
-        assert result == "한시간 후"
+        assert self.locale._format_relative("1초", "second", 1) == "1초 후"
+        assert self.locale._format_relative("2초", "seconds", 2) == "2초 후"
+        assert self.locale._format_relative("1분", "minute", 1) == "1분 후"
+        assert self.locale._format_relative("2분", "minutes", 2) == "2분 후"
+        assert self.locale._format_relative("한시간", "hour", 1) == "한시간 후"
+        assert self.locale._format_relative("2시간", "hours", 2) == "2시간 후"
+        assert self.locale._format_relative("하루", "day", 1) == "내일"
+        assert self.locale._format_relative("2일", "days", 2) == "모레"
+        assert self.locale._format_relative("3일", "days", 3) == "글피"
+        assert self.locale._format_relative("4일", "days", 4) == "그글피"
+        assert self.locale._format_relative("5일", "days", 5) == "5일 후"
+        assert self.locale._format_relative("1주", "week", 1) == "1주 후"
+        assert self.locale._format_relative("2주", "weeks", 2) == "2주 후"
+        assert self.locale._format_relative("한달", "momth", 1) == "한달 후"
+        assert self.locale._format_relative("2개월", "months", 2) == "2개월 후"
+        assert self.locale._format_relative("1년", "year", 1) == "내년"
+        assert self.locale._format_relative("2년", "years", 2) == "내후년"
+        assert self.locale._format_relative("3년", "years", 3) == "3년 후"
 
     def test_format_relative_future(self):
-
-        result = self.locale._format_relative("한시간", "hour", -1)
-
-        assert result == "한시간 전"
+        assert self.locale._format_relative("1초", "second", -1) == "1초 전"
+        assert self.locale._format_relative("2초", "seconds", -2) == "2초 전"
+        assert self.locale._format_relative("1분", "minute", -1) == "1분 전"
+        assert self.locale._format_relative("2분", "minutes", -2) == "2분 전"
+        assert self.locale._format_relative("한시간", "hour", -1) == "한시간 전"
+        assert self.locale._format_relative("2시간", "hours", -2) == "2시간 전"
+        assert self.locale._format_relative("하루", "day", -1) == "어제"
+        assert self.locale._format_relative("2일", "days", -2) == "그제"
+        assert self.locale._format_relative("3일", "days", -3) == "그끄제"
+        assert self.locale._format_relative("4일", "days", -4) == "4일 전"
+        assert self.locale._format_relative("1주", "week", -1) == "1주 전"
+        assert self.locale._format_relative("2주", "weeks", -2) == "2주 전"
+        assert self.locale._format_relative("한달", "momth", -1) == "한달 전"
```

Comment:
misspelled month also?

Reply:
it will be funny if this actually works hah

Reply:
"momth" is cleary my fault. I'll fix it soon.
This actually works because of inherited `Locale._format_relative` method.
It just returns 1st parameter that presents `humanized` string without reference 2nd `timeframe`.